documentation/src/docs/asciidoc/extensions.adoc

```diff
@@ -178,25 +99,27 @@ resolved by _name_, _type_, _annotation_, or any combination thereof. For concre
 consult the source code for `{CustomTypeParameterResolver}` and
 `{CustomAnnotationParameterResolver}`.
 
+
 === Test Lifecycle Callbacks
 
 The following interfaces define the APIs for extending tests at various points in the
 test execution lifecycle. Consult the Javadoc for each of these in the
-`{extension-api-package}` package.
+`{extension-api-package}` package for further details.
 
-* `BeforeEachExtensionPoint`
-* `AfterEachExtensionPoint`
-* `BeforeAllExtensionPoint`
-* `AfterAllExtensionPoint`
+* `BeforeEachCallback`
+* `AfterEachCallback`
+* `BeforeAllCallback`
+* `AfterAllCallback`
```

Comment:
The two new extension points (`BeforeTestMethodCallback` and `AfterTestMethodCallback`) are missing here, right?